config/v1/types_cluster_version.go

```diff
@@ -73,11 +73,16 @@ type ClusterVersionSpec struct {
 // progress, or is failing.
 // +k8s:deepcopy-gen=true
 type ClusterVersionStatus struct {
-	// current is the version that the cluster will be reconciled to. This
-	// value may be empty during cluster startup, and then will be set whenever
-	// a new update is being applied. Use the conditions array to know whether
-	// the update is complete.
-	Current Update `json:"current"`
+	// desired is the version that the cluster is reconciling towards.
+	// If the cluster is not yet fully initialized desired will be set
+	// with the information available, which may be a payload or a tag.
+	Desired Update `json:"desired"`
+
+	// history contains a list of the most recent versions applied to the cluster.
+	// This value may be empty during cluster startup, and then will be updated
+	// when a new update is being applied. Use the conditions array to know whether
+	// the update is complete. Only a limited amount of update history is preserved.
+	History []UpdateHistory `json:"history"`
```

Comment:
document the ordering.   Most recent to least recent or least recent to most recent